snyk-import/README.md

```diff
@@ -0,0 +1,36 @@
+# Snyk Import Extension
+
+This extension imports Snyk projects into Phylum by retrieving dependency
+information from the Snyk API and submitting the dependencies as a new analysis
+job to a Phylum project of the same name.
+
+## Installation
+
+Clone the repository and install the extension via the Phylum CLI.
+
+```console
+git clone https://github.com/phylum-dev/community-extensions
+phylum extension install community-extensions/snyk-import/
+```
+
+## Running
+
+To import all of your current Snyk projects
```

Comment:
This is pretty nitpicky but the punctuation before code blocks in this readme is very inconsistent. For the first one you use a full stop, then nothing, then a colon. I'd assume you omitted the colon/full stop here since the next sentence starts with `Or`? If so, then it shouldn't be capitalized.

Personally I'd probably just use colons for them all.